subprojects/packagefiles/libupnp/meson.build

```diff
@@ -0,0 +1,292 @@
+project('pupnp', 'c',
+  version : '1.14.12',
+  default_options : [ 'c_std=gnu99', 'warning_level=1' ]
+)
+
+libupnp_incdir = include_directories([ '.', 'upnp/inc', 'upnp/src/inc', 'upnp/src/threadutil' ])
+libixml_incdir = include_directories([ 'ixml/inc', 'ixml/src/inc' ])
+
+libixml = static_library('ixml',
```

Comment:
Why is this specifically a static library, especially considering it is installed?

Reply:
Because nothing besides libupnp uses it.

Reply:
Then in that case, why is it installed? And why does it also install public headers?

Reply:
Beats me.